portfolio/src/main/webapp/book-detail/book-detail.component.js

```diff
@@ -53,6 +54,13 @@ angular.module('bookDetail').component('bookDetail', {
         var fieldContentArray = [title, genre, categories, author, language, description, infoLink, pageCount, publishedDate, publisher, maturityRating];
         var fieldsString;
         var index = 0;
+
+        console.log("testing file name:");
+        if (new String(title).toLowerCase().valueOf() != new String(info.title).toLowerCase().valueOf()) {
+            console.log("fail!");
+            console.log(title);
+            console.log(info.title);
+        }
```

Comment:
Looks like this code is only for testing, and doesn't need to be checked in?

Reply:
Wanted to leave it in for when we expand to the 300ish books in the dataset (~ this week) so that we can identify any mismatches!